### PR TITLE
MLR 323 - Duplicate Normalized Station Name Validation

### PR DESCRIPTION
**Pull request by cschroed-usgs**
This feature required wiring the Validator up to the CRU service.

TODO: 
 * client-side load balancing
 * exponential backoff retry
 * integration tests

Am I on the right track?

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -19,7 +19,8 @@
     application.config['JWT_PUBLIC_KEY'] = resp.json()['value']
     application.config['JWT_ALGORITHM'] = 'RS256'
 
-error_validator = ErrorValidator(application.config['SCHEMA_DIR'], application.config['REFERENCE_FILE_DIR'])
+error_validator = ErrorValidator(application.config['SCHEMA_DIR'], application.config['REFERENCE_FILE_DIR'],
+                                 application.config['CRU_SERVICE_URL'])
```
Comment (cschroed-usgs) [inline app.py]: Currently this takes in a single URL from the `.env` file. Is that good enough for now?

File: mlrvalidator/validators/tests/test_cru_validator.py [hunk 2]
```diff
@@ -0,0 +1,282 @@
...
+
+        existing_station0 = {
+            **base_site,
+            "projectNumber": "2",
+        }
+        existing_station1 = {
+            **base_site,
+            "projectNumber": "3",
+        }
+        valid = self.instance._validate_normalized_station_name(station, [existing_station0, existing_station1])
```
Comment (kmschoep-usgs) [inline mlrvalidator/validators/tests/test_cru_validator.py]: this implies that multiple records will come back with the same siteNumber and agencyCode, which is impossible with the existing constraint on the table.

Comment (cschroed-usgs) [reply mlrvalidator/validators/tests/test_cru_validator.py]: Good catch! I will correct it.

**Comment by cschroed-usgs** [comment]
> Make sure to update the changelog with this new functionality (and other bug fixes).

Thanks! Done!

**Pull request merged**